Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.h

```diff
@@ -64,7 +64,7 @@ class CDMInstanceOpenCDM final : public CDMInstance {
         static Ref<Session> create(const media::OpenCdm& source, Ref<WebCore::SharedBuffer>&& initData);
 
         bool isValid() const { return m_isValid; }
-        const std::string& message() const { return m_message; }
+        Ref<SharedBuffer> message() const { ASSERT(m_message); return Ref<SharedBuffer>(*m_message.get()); }
```

Comment:
Wonder what you think to making this a RefPtr and letting the caller deal with the possibility of an empty message?

Reply:
From what I see that message is populated at construction and should not null (it could be empty though). I keep a `RefPtr` because I don't know how to construct the object in the constructor and keep a `Ref<SharedBuffer>` instead of a `RefPtr` as it will complain at compiling time because the `Ref<SharedBuffer>` cannot be initialized during allocation.